proto/apl.proto

```diff
@@ -70,16 +70,18 @@ message APLAction {
         APLActionCancelAura cancel_aura = 10;
         APLActionTriggerICD trigger_icd = 11;
         APLActionItemSwap item_swap = 17;
+		APLActionMove move = 18;
+        APLActionMoveDuration move_duration = 22;
 
         // Class or Spec-specific actions
-        APLActionCatOptimalRotationAction cat_optimal_rotation_action = 18;
+        APLActionCatOptimalRotationAction cat_optimal_rotation_action = 19;
 
         // Internal use only, not exposed in UI.
-        APLActionCustomRotation custom_rotation = 19;
+        APLActionCustomRotation custom_rotation = 21;
```

Comment:
Changing proto indices is a bad patter as it breaks any saved data of users. Always add new actions to the next index without changing other actions/values indices